custom/icds_reports/static/js/directives/indie-map/indie-map.directive.js

```diff
@@ -343,9 +343,12 @@ function IndieMapController($scope, $compile, $location, $filter, storageService
         return html;
     };
 
-    vm.closePopup = function () {
-        var popup = d3.select("#locPopup");
-        popup.classed("hidden", true);
+    vm.closePopup = function (e) {
+        // checking if click event is triggered on map
+        if (e.target.tagName !== 'path') {
```

Comment:
confirming you tested this doesn't mess with anything on web dashboard?

Reply:
Yes, I have tested